doc/TROUBLESHOOTING.md

```diff
@@ -205,3 +205,39 @@ java.io.IOException: Invalid encoded sequence encountered: 3D 3D 5B 4A 45 4E 4B
 ### Use Remote root directory in a no C: drive
 
 The default configuration assumes the Remote root directory in `C:` drive, so the agent command launch will fail if the Remote root directory is in another drive. You can change the Remote root directory drive by using `Prefix Start Agent Command`, if you set `Prefix Start Agent Command` to `cd /d D:\ &&` you would change to the drive `D:` before to enter in the Remote root directory.
+
+
+### PEM problem: it is of unknown type
+
+When the Key format imported in credentials is not supported bu trilead-ssh2 library, you see the following error
+
+```
+ERROR: Server rejected the 1 private key(s) for jenkins  (credentialId:jenkins-test-ssh-key/method:publickey)
+ERROR: Failed to authenticate as jenkins  with credential=jenkins-test-ssh-key
+java.io.IOException: Publickey authentication failed.
+	at com.trilead.ssh2.auth.AuthenticationManager.authenticatePublicKey(AuthenticationManager.java:358)
+	at com.trilead.ssh2.Connection.authenticateWithPublicKey(Connection.java:472)
+	at com.cloudbees.jenkins.plugins.sshcredentials.impl.TrileadSSHPublicKeyAuthenticator.doAuthenticate(TrileadSSHPublicKeyAuthenticator.java:109)
+	at com.cloudbees.jenkins.plugins.sshcredentials.SSHAuthenticator.authenticate(SSHAuthenticator.java:436)
+	at com.cloudbees.jenkins.plugins.sshcredentials.SSHAuthenticator.authenticate(SSHAuthenticator.java:473)
+	at hudson.plugins.sshslaves.SSHLauncher.openConnection(SSHLauncher.java:863)
+	at hudson.plugins.sshslaves.SSHLauncher$1.call(SSHLauncher.java:435)
+	at hudson.plugins.sshslaves.SSHLauncher$1.call(SSHLauncher.java:422)
+	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
+	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
+	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
+	at java.lang.Thread.run(Thread.java:748)
+Caused by: java.io.IOException: PEM problem: it is of unknown type
+	at com.trilead.ssh2.crypto.PEMDecoder.decodeKeyPair(PEMDecoder.java:500)
+	at com.trilead.ssh2.auth.AuthenticationManager.authenticatePublicKey(AuthenticationManager.java:292)
+	... 11 more
+```
+
+Not all PEM formats are supported, compare the header of your key with the following supported formats:
+* `-----BEGIN OPENSSH PRIVATE KEY-----`
+* `-----BEGIN RSA PRIVATE KEY-----`
+* `-----BEGIN EC PRIVATE KEY-----`
+* `-----BEGIN DSA PRIVATE KEY-----`
+In case your key format is not supported you can use `openssl` to convert it.
```

Comment:
```suggestion
In case your key format is not supported you can use `openssl` or `ssh-keygen` to convert it to the SSH protocol version 2 format (RFC4716).
```